app/src/main/java/com/example/househomey/form/ItemFormFragment.java

```diff
@@ -179,4 +199,63 @@ public void afterTextChanged(Editable editable) {
             }
         });
     }
+
+    /**
+     * Initializes photo components for the fragment (photo adapter,
+     * and image picker dialog) as well as sets this fragment to the components' listeners.
+     *
+     * @param rootView The root view of the fragment.
+     */
+    private void initAddImageHandler(View rootView) {
+        // Set up adapter for gallery and listener for adding photos
+        photoAdapter = new PhotoAdapter(getContext(), photoUris);
+        ((RecyclerView) rootView.findViewById(R.id.add_photo_grid)).setAdapter(photoAdapter);
```

Comment:
I would say no. RecyclerView is way more complicated. The reason I needed it was:
1. GridView (my initial choice) is scrollable, and therefore can't be placed inside another ScrollView. This wouldn't work for the Add/Edit pages, which scroll. But our ListView is the only thing that scrolls on Home, so that's fine.
2. RecyclerView allows you to have different ViewTypes in the same adapter. So it allowed me to add the Add Button in-line with the grid, like how we wanted it in the UI. Whereas our ListView only has one ViewType.

So I would just leave ListView as is, unless we need diff list element views (ex: photo vs non-photo list item) later